packages/make-styles/src/runtime/utils/test/snapshotSerializer.ts

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Let's move out it from `/runtime/utils` to avoid accidental bundling into in a real code